Source/WebCore/platform/network/cocoa/CookieCocoa.mm

```diff
@@ -97,7 +97,7 @@ static NSHTTPCookieStringPolicy _Nullable nsSameSitePolicy(Cookie::SameSitePolic
 {
     switch (policy) {
     case Cookie::SameSitePolicy::None:
-        return nil;
+        return @"none"; // FIXME: This should use a string constant (rdar://135227277)
```

Comment:
Should coreSameSitePolicy also check for "none"?

Reply:
None will be used if there is no match, but I should will check “none” so we don’t hit the debug assertion in that function.